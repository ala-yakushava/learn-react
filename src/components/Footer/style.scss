@import "../../styles/theme";

.Footer {
  display: flex;
  justify-content: center;
  margin-top: auto;
  padding: 20px;
  background-color: $dark-grey;
}
