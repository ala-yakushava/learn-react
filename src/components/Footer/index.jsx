import React from 'react';

import './style.scss';
import Logo from '../Logo';

const Footer = () => {
  return (
    <footer className="Footer">
      <Logo />
    </footer>
  );
}

export default Footer;
