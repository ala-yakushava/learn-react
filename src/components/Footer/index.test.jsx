import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './index';

describe('Components - Footer', () => {
  test('should match snapshot', () => {
    const tree = renderer
      .create(<Footer />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
