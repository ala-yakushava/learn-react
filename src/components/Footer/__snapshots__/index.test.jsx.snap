// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Footer should match snapshot 1`] = `
<footer
  className="Footer"
>
  <p
    className="Logo"
  >
    <b>
      netflix
    </b>
    roulette
  </p>
</footer>
`;
